data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,56 @@ mission "Remnant: Scent of Rain 1"
 			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
 			label scanningpayment
 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
+
+
+
+mission "Remnant: Will Not Someone Please Think Of The Children"
+	minor
+	landing
+	name "Sympathy For The Korath"
+	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
+	source
+		government "Remnant"
+	waypoint "Vaticanus"
+	to offer
+		random < 30
+		"remnant: broken jump drive count" >= 10
+		or
+			and
+				has "Remnant: Cognizance 25: done"
+				not "flagship planet: Ssil Vida"
+			has "event: remnant: cognizance calm"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first>, I have a sensitive request for you."`
+			`	"Our scouts are reporting a stranded Korath Kas'lor Ik 582 in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
+			`	"Would you be up for disabling this Korath Kas'lor Ik 582, so the threat is eliminated while preserving the lives of the innocents on board?" His signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
+			choice
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have no intentions of risking my neck for some Korath."`
+					goto rathernot
+			`	"I am grateful for your understanding and willingness to help here. Together we can ensure everyone gets home safely."`
+				accept
+			label rathernot
+			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
+				decline
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
```

Comment:
why heroic? why not timid?

Reply:
Consider what they're defending.  A ship with many civilians on board, equipped with long-range artillery?  The `olofezes I think should do whatever it takes.... especially since they can't exactly flee anywhere.

Reply:
timid means they stay near the ship and thus are **much** more efficient at defending it

Reply:
it seems to work
heroic is the worst thing to give to a fighter it needs to stay near the flagship to get repairs and actually defend it

Reply:
Cool.  Just mind you this is the mothership not the fighters, I'll change that on them.  They have swarming to keep them near the ship, too.

Reply:
I did tell you that I thought swarming ships shoot their own? Anyhow if that's still the case its a separate thing that needs fixing